configs/openmrs_config/concepts/physical_examination.csv

```diff
@@ -19,7 +19,7 @@ b5aead5b-e0ed-4462-8d38-44cc0ab63dfc,,,Monofilament-Abnormal,Monofilament មិ
 c08665e0-5f0a-4d23-886d-d14e8e1bf40d,,,Numbness,ស្ពឹក,Numbness,ស្ពឹក,,,Misc,N/A,,,,
 978422b8-a953-44c4-8dfb-ca7128d1e4ca,,,Wounds,របួស,Wounds,របួស,,,Misc,N/A,,,,
 f10be2a3-fd0d-43d8-8f6c-1f3d6d779044,,,Pain,ចុករោយ,Pain,ចុករោយ,,,Misc,N/A,,,,
-11a2da88-7235-4182-b808-99ba2fb4a1a5,,,Foot problem,បញ្ហាជើង,Foot problem,បញ្ហាជើង,,,Question,Coded,Numbness;Wounds;Pain,,,
+c01f876c-0a08-47c6-a003-e3dd22098634,,,problems:,បញ្ហាជើង:,problems:,បញ្ហាជើង:,,,Question,Coded,Numbness;Pain;Wounds,,,
```

Comment:
Why are we changing the concept name from `Foot problem` to `problems`?

Reply:
The `foot problem` concept was existing as a coded question and as a single concept answer which was causing conflicts. I didnt change the khmer to avoid altering the requirements

Reply:
[
![KH-255-FOOT AND TB](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/6fa80d5a-a019-46ff-bf84-8c400432cce5)
](url)

Reply:
@kdaud please note

Reply:
@kazlaw when we change the concept name from `Foot problem` to `problem`, will not make meaning to the user. 

> The foot problem concept was existing as a coded question and as a single concept answer which was causing conflicts.

Which conflict is showing up?

Reply:
This has been worked on , kindly test and we resolve this

![Works well on Gitpod](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/08b1527a-819f-47c5-942d-2b2f8d76fcb0)

Reply:
@kdaud please have a look